server/logout.js

```diff
@@ -0,0 +1,6 @@
+export const logout = (req, res) => {
+  res.clearCookie('authCookie');
+  return res.status(200).redirect('/loginPage');
+
+  //for some reason I can't get it to work without the redirect
```

Comment:
You need to send it. Our convention is to send json, so like `.json('logged out')`